handler/core/src/main/java/io/knotx/fragments/handler/action/cache/InMemoryCacheActionFactory.java

```diff
@@ -0,0 +1,38 @@
+package io.knotx.fragments.handler.action.cache;
```

Comment:
Please provide a license header.
I wonder why the rat plugin didn't found that.
Please also check other files for the missing license header.